src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -38,6 +41,19 @@ public virtual void ConfigureServices(IServiceCollection services)
                 .AddSqlServer(Configuration)
                 .AddBackgroundWorkers();
 
+            services.AddMvc()
```

Comment:
Should the versioning be configured here or in the API project?

Reply:
It has to be in the Startup class I think. I'm not sure if there's a way to move this to the API project

Reply:
Our APIs after this change will leverage verisoning as part of their respective routes. As such, we should make API verisoning a requirement of our API project and in turn register the appropriate API versioning services at the same time we register our "MVC" services. Currently this registration happens in the `AddDicomServer` method used above, so I think we should add any versioning service registration there too.

Furthermore, it would have the added benefit of re-using the same code between OSS and PAAS.

Reply:
Ah thanks! I didn't think to look inside `AddDicomServer ` to see what it was doing. I agree it makes sense to move the configuration there. 

& to clarify - the API project is used in PAAS, and the Web project is not? Is that correct?

Reply:
Yeah, OSS and PAAS have their own respective "web" projects that both leverage the API project for controllers, services, etc

Reply:
I've moved the configuration to AddDicomServer